File: python/simple-interest.py
def calculate_simple_interest(principal, time, rate):
    simple_interest = (principal * time * rate) / 100
    print(f"Simple Interest: {simple_interest:.2f}")

# Taking inputs
principal = float(input("Enter Principal Amount: "))
time = float(input("Enter Time (in years): "))
rate = float(input("Enter Rate of Interest: "))

# Calculations
calculate_simple_interest(principal, time, rate)

File: C/simple_interest.c
#include <stdio.h>

void calculate_simple_interest(float principal, float time, float rate) {
    float simple_interest = (principal * time * rate) / 100;
    printf("Simple Interest: %.2f\n", simple_interest);
}

int main() {
    float principal, time, rate;

    // Display heading
    printf("========================================\n");
    printf("        SIMPLE INTEREST CALCULATOR      \n");
    printf("========================================\n");

    printf("Enter Principal Amount: ");
    scanf("%f", &principal);
    printf("Enter Time (in years): ");
    scanf("%f", &time);
    printf("Enter Rate of Interest: ");
    scanf("%f", &rate);

    calculate_simple_interest(principal, time, rate);

    return 0;
}
